WordPress/Classes/Services/SuggestionService.m

```diff
@@ -91,23 +82,84 @@ - (void)updateSuggestionsForSiteID:(NSNumber *)siteID
     }];
 }
 
+- (Autocompleter *)retrieveAutocompleterForSiteID:(NSNumber *)siteID
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+
+    NSFetchRequest *request = Autocompleter.fetchRequest;
+    [request setPredicate:[NSPredicate predicateWithFormat:@"siteID == %@", siteID]];
+    NSError *error;
+    NSArray *sites = [context executeFetchRequest:request error:&error];
+    if (error) {
+        DDLogError(@"Could not fetch Autocompleter for site %@, error", siteID, error);
```

Comment:
```suggestion
        DDLogError(@"Could not fetch usernames for site %@, error", siteID, error);
```

Reply:
Or perhaps, "username suggestions" or "user suggestions"

Reply:
Thanks! I'll improve this.

Reply:
Marking this as resolved because the latest commits got rid of this code.